Person.java

```diff
@@ -0,0 +1,16 @@
+public class Person{
+  String type,name,email;
```

Comment:
`private`, `protected`, or `public`? Would I want to be able to use `person.type` and override it? What about the name / email?